src/v/raft/types.cc

```diff
@@ -121,7 +120,6 @@ T read_one_varint_delta(iobuf_parser& in, const T& prev) {
     auto dst = varlong_reader<T>(in);
     return prev + dst;
 }
-} // namespace internal
 } // namespace
```

Comment:
i don't even think the anonymous namespace is needed. the templates aren't exported, and neither are the types.

for future reference, it's a good commit to pluck out as a one-off pr.

Reply:
Good point. DOne